client/query_test.go

```diff
@@ -34,7 +34,7 @@ func (s *IntegrationTestSuite) TestQueryABCIHeight() {
 			name:      "empty request height and context height - use latest height",
 			reqHeight: 0,
 			ctxHeight: 0,
-			expHeight: 4,
+			expHeight: 5,
```

Comment:
There doesn't seem to be any relevant changes, is there a reason you changed the values?

Reply:
In many cases, an error occurs when it is 4. I fixed it because the response is coming from 5 because it is waiting somewhere.